paddle/fluid/framework/operator.cc

```diff
@@ -1143,14 +1182,17 @@ Scope* OperatorWithKernel::PrepareData(
       // In the inference scenerio, the scopes will be reused across the
       // batches, so the `new_scope` here will result in GPU memroy explosion
       // over the  running of operators.
-      // We use a thread_local cache to fix that issue, the key in the cache is
+      // We use a thread_local cache to fix that issue, the key in the cache
+      // is
```

Comment:
line1186, line1193, line1220 have some formatting problem, please fix in next PR.

Reply:
ok